src/lib/seam/connect/seam-http-request.ts

```diff
@@ -0,0 +1,127 @@
+import { serializeUrlSearchParams } from '@seamapi/url-search-params-serializer'
+import type { Method } from 'axios'
+
+import type { Client } from './client.js'
+import type { SeamHttpRequestOptions } from './options.js'
+import { resolveActionAttempt } from './resolve-action-attempt.js'
+import { SeamHttpActionAttempts } from './routes/index.js'
+
+interface SeamHttpRequestParent {
+  readonly client: Client
+  readonly defaults: Required<SeamHttpRequestOptions>
+}
+
+export interface SeamHttpRequestConfig<TBody, TResponseKey> {
```

Comment:
Is there a clear use case to export this?
```suggestion
interface SeamHttpRequestConfig<TBody, TResponseKey> {
```

Reply:
Not imo